W/Wayland_protocols/build_tarballs.jl

```diff
@@ -3,12 +3,12 @@
 using BinaryBuilder
 
 name = "Wayland_protocols"
-version = v"1.23"
+version = v"1.24"
```

Comment:
There is also 1.25 now

Reply:
https://github.com/JuliaPackaging/Yggdrasil/pull/4494